.github/actions/dockerhub/Dockerfile.debian-source

```diff
@@ -0,0 +1,21 @@
+FROM rust:bullseye as build
+
+ARG STACKS_NODE_VERSION="No Version Info"
+ARG GIT_BRANCH='No Branch Info'
+ARG GIT_COMMIT='No Commit Info'
+ARG BUILD_DIR=/build
+ARG TARGET=x86_64-unknown-linux-gnu
+WORKDIR /src
+
+COPY . .
+
+RUN --mount=type=tmpfs,target=${BUILD_DIR} cp -R /src/. ${BUILD_DIR}/ \
```

Comment:
You're going to need ~8GB disk space to comfortably build the release.  Is this guaranteed to be the case here?

Reply:
i'll have to RTFM here, i don't know if that will always be the case. in testing, i haven't seen this error but i'll look into it more. At a minimum, it can be reverted to using bind-mount disk access which is how things are currently being built.

Reply:
The standard GH runners have [14GB of disk space](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) so it should be fine. If the space required is expected to grow over time, we might have to switch to a hosted runner some time down the road.

Reply:
leaving this as-is for now. we can adjust in the future if we hit disk limits in the future